roles/openstack_create_server/vars/main.yml

```diff
@@ -2,7 +2,7 @@
 # Rough equivalents of the Amazon ec2 flavors.
 ec2_flavors:
   # https://aws.amazon.com/ec2/previous-generation/
-  - { name: m1.small, vcpu: 1, memory: 1740, disk: 64 }
+  - { name: m1.small, vcpu: 1, memory: 1740, disk: 68 }
```

Comment:
Why 68? Safety of 2GBs?

Reply:
Yes it is not clear to me if these units are in GiB (67584) or GB (66000) . I do not want any problems if I misunderstood the units.